tests/queries/0_stateless/01487_distributed_in_not_default_db.sql

```diff
@@ -25,10 +25,6 @@ CREATE TABLE d AS t ENGINE = Distributed(test_cluster_two_shards_different_datab
 USE test_01487;
 DROP DATABASE test_01487;
 
--- After the default database is dropped QueryAnalysisPass cannot process the following SELECT query.
--- That query is invalid on the initiator node.
-set allow_experimental_analyzer = 0;
-
 SELECT * FROM main_01487.d WHERE value IN (SELECT l.value FROM l) ORDER BY value;
```

Comment:
This is invalid on initiator node SELECT query. We are not supporting it anymore.